app/queries/send-notification.js

```diff
@@ -0,0 +1,20 @@
+const { v4: uuidv4 } = require('uuid')
+const NotifyClient = require('notifications-node-client').NotifyClient
+
+const sendNotification = async (message) => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY)
+
+  await notifyClient.sendEmail(
+    process.env.NOTIFY_TEMPLATE_ID,
+    'rana.salem@defra.gov.uk',
```

Comment:
Could change this to be an ENV variable instead so you dont get spammed when others use the service

Reply:
Done, new env var is `NOTIFY_TEST_EMAIL` (see `docker-compose.yaml`).